### PR TITLE
bpo-40558: update CONTRIBUTING.rst docs

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue40558

### CODE REVIEW

**Comment by shihai1991** [comment]
@zware Hi, Zachary. Pls take a look if you have free time. 
I have added the 3.9's link of buildbot.

File: .github/CONTRIBUTING.rst [hunk 1]
```diff
@@ -8,17 +8,17 @@ Build Status
...
+- 3.8
 
-  + `Stable buildbots <http://buildbot.python.org/3.6.stable/>`_
+  + `Stable buildbots <http://buildbot.python.org/3.8.stable/>`_
 
-- 2.7
+- 3.7
 
-  + `Stable buildbots <http://buildbot.python.org/2.7.stable/>`_
+  + `Stable buildbots <http://buildbot.python.org/3.7.stable/>`_
```
Comment (zware) [inline .github/CONTRIBUTING.rst:21]: I wonder if we should simplify this to just link to the 3.x stable set and mention that you can adjust the URL manually (replace the `x` with the minor version of your choice) to get the stable set for a maintenance branch.

**Review by zware — APPROVED** [review]
LGTM, with a suggestion you can take or leave in a new PR.

File: .github/CONTRIBUTING.rst (hunk 1 above)
Comment (shihai1991) [reply .github/CONTRIBUTING.rst:21]: Good idea : )
I will take a look(I am sure we can do it or not now)

**Pull request merged**